package.json

```diff
@@ -7,7 +7,7 @@
 	"repository": "yeoman/doctor",
 	"main": "lib/index.js",
 	"bin": {
-		"yodoctor": "lib/cli.js"
+		"yo-doctor": "lib/cli.js"
```

Comment:
Hej @LitoMore,

this should be the following to prevent breaking backward compatibility:

```javascript
"bin": {
  "yodoctor": "lib/cli.js",
  "yo-doctor": "lib/cli.js"
},
```

I know I’m late to the party since this is already merged. Are you interested in opening another PR? Otherwise I would »fix« this.

Cheers, Michael

Reply:
@mischah Sure. I will make a pull request to revert this change.